build/CoreFoundation/dll/CoreFoundation.vcxproj

```diff
@@ -152,7 +152,7 @@
       <IgnoreAllDefaultLibraries>false</IgnoreAllDefaultLibraries>
       <GenerateWindowsMetadata>false</GenerateWindowsMetadata>
       <ModuleDefinitionFile>CoreFoundation.def</ModuleDefinitionFile>
-      <AdditionalDependencies>mincore.lib;libxml2.lib;icudt.lib;icuin.lib;icuuc.lib;libdispatch.lib;icudata.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
Yeah, I think that'd be the case. Also reminder to everyone that eventually we'll consume this directly from windows, meaning we'll no longer take the hit.

---
In reply to: [71459710](https://github.com/Microsoft/WinObjC/pull/668#discussion_r71459710) [](ancestors = 71459710,71421348,71421042,71389811)